src/laravel/README.md

```diff
@@ -24,11 +24,16 @@ APP_RUNTIME=Runtime\Laravel\Runtime
 
 use Illuminate\Contracts\Http\Kernel;
 
+define('LARAVEL_START', microtime(true));
+
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
-return function () {
-    define('LARAVEL_START', microtime(true));
-    $app = require_once __DIR__ . '/../bootstrap/app.php';
+return function (): Kernel {
+    static $app;
+
+    if (null === $app) {
+        $app = require dirname(__DIR__).'/bootstrap/app.php';
+    }
```

Comment:
when using amphp, react, roadrunner .. etc, this means that the application shares the same state between all requests, is that considered okay?

Reply:
Yes. That is the entire point. :)

Reply:
I think I got the warning when running this front controller with `artisan`.